static/js/operation.js

```diff
@@ -123,75 +123,76 @@ function parseAssembly(text, data, minContigLength) {
   var cols = ['id', 'length', 'gc', 'coverage']; // the information available in the contig titles
   var df = [];
 
-  var id = 0;
+  var id = NaN;
   var length = 0;
   var gc = 0;
   var coverage = 0;
-  var currentContigLength = 0;
-  var gcCount = 0;
 
   for (var i = 0; i < lines.length; i++) {
     var line = lines[i];
-    if (line.search(/^>{1}/gm) === 0) { // Checking if the current line is a contig title
-        [id, length, coverage] = parseContigTitles(line, format);
-        if (length <= minContigLength) 
-          continue;
-
-        // append dataframe with contig title information
-        df.push([id, length, , coverage]) // gc not calculated yet
-        gcCount = 0;
-        currentContigLength = 0;
+    if (line.charAt() === '>') { // Checking if the current line is a contig title
+        if (id !== NaN && length > minLen) {
```

Comment:
I think this `length > minLen` should be `length >= minLen`. For example, if threshold is 1000, then a contig of 1000 bp should be included.